test/runtests.jl

```diff
@@ -28,18 +29,18 @@ form = DataFrame(Any[[0.1,0.3,0.5,0.6,0.7,0.9],[0.086,0.269,0.446,0.538,0.626,0.
     @test isapprox(r²(lm1), 0.9990466748057584)
     @test adjr²(lm1) == adjr2(lm1)
     @test isapprox(adjr²(lm1), 0.998808343507198)
-    @test isapprox(aic(lm1), -36.409684288095946)
-    @test isapprox(aicc(lm1), -24.409684288095946)
-    @test isapprox(bic(lm1), -37.03440588041178)
+    @test isapprox(StatsBase.aic(lm1), -36.409684288095946)
```

Comment:
`StatsBase.` shouldn't be needed AFAIK?

Reply:
Thanks for catching that.  It's vestigial.  Initially I wasn't including `StatsBase` and using fully qualified names instead.